.github/workflows/docker-build.yml

```diff
@@ -4,10 +4,11 @@ on:
   push:
     branches:
       - "**"
```

Comment:
This should be main, then you don't need to have the `github.ref` check down below. 
```suggestion
      - main
```

Reply:
testing just `push` event due to the `if` condition filtering